src/positions/Positions.tsx

```diff
@@ -0,0 +1,147 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { ActivityIndicator, Image, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { positionsSelector, positionsStatusSelector } from 'src/positions/selectors'
+import { BaseToken, Position, Token } from 'src/positions/types'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { tokensByAddressSelector } from 'src/tokens/selectors'
+import { Currency } from 'src/utils/currencies'
+
+function getBaseTokens(tokens: Token[]): BaseToken[] {
+  return tokens.flatMap((token) => {
+    if (token.type === 'base-token') {
+      return [token]
+    } else {
+      return getBaseTokens(token.tokens)
+    }
+  })
+}
+
+function PositionItem({ position }: { position: Position }) {
+  const tokens = useSelector(tokensByAddressSelector)
+
+  const baseTokens = getBaseTokens(position.tokens)
+  const baseTokenImages = baseTokens
+    .map((token) => {
+      const tokenInfo = tokens[token.address]
+      return tokenInfo ? tokenInfo.imageUrl : undefined
+    })
+    .filter((image) => image !== undefined)
+
+  const balanceInDecimal =
+    position.type === 'contract-position' ? undefined : new BigNumber(position.balance)
+  const balanceUsd =
+    position.type === 'contract-position'
+      ? new BigNumber(position.balanceUsd)
+      : new BigNumber(position.balance).multipliedBy(position.priceUsd)
+
+  return (
+    <View style={styles.tokenContainer}>
+      <View style={styles.row}>
+        <View style={{ flexDirection: 'row-reverse' }}>
+          {baseTokenImages.reverse().map((image, index) => (
```

Comment:
Just curious, why do we need the `reverse()` here?

Reply:
It's needed (along with `row-reverse`) because I wanted the first image on the left to be on top of the images on the right, with the overlap:
<img width="90" alt="Screenshot 2023-05-05 at 12 20 09" src="https://user-images.githubusercontent.com/57791/236434310-6f89128b-6de4-4407-ad9d-6d67078c05d4.png">

By default for displaying `[MOO, CELO]`, with the default `row` layout we'd have:
<img width="93" alt="Screenshot 2023-05-05 at 12 24 01" src="https://user-images.githubusercontent.com/57791/236434410-421aa6ed-f1b3-49cf-baa0-5d8c961f6a4a.png">
i.e. the images drawn last are displayed on top.
`CELO` is last so displayed on top of `MOO`

If we apply `row-reverse` we get this:
<img width="86" alt="Screenshot 2023-05-05 at 12 24 38" src="https://user-images.githubusercontent.com/57791/236434550-615e5752-2cb5-4f13-9a7d-2e0e561cbe0d.png">
Again `CELO` is last so displayed on top of `MOO`, but it started drawing them from the right.

So to get `MOO` on top of `CELO`, with `MOO` on the left, we need to `reverse()` the array.